### PR TITLE
Fix/test e2e commands

### PR DESCRIPTION
**Pull request by portly-halicore-76**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: .github/workflows/ci-build.yaml [hunk 1]
```diff
@@ -86,6 +86,44 @@ jobs:
...
+            count=$((count+1))
+          done
+      - name: Run E2E testsuite
         timeout-minutes: 20
-        run: make test-e2e-against-kind
\ No newline at end of file
+        run: |
+          set -x
+          make test-e2e
+          killall goreman run stop-all || echo "goreman trouble"
```
Comment (hikarukin) [inline .github/workflows/ci-build.yaml]: Would be nice to have a slightly more accurate message.

File: docs/development-guide/e2e-tests.md [hunk 2]
```diff
@@ -10,7 +10,10 @@ date: 2024-08-21
 
 ## Running the tests
 
-Run the tests using the following command: `go test -v ./test/e2e`
+1. Have a k8s or OpenShift cluster running
```
Comment (hikarukin) [inline docs/development-guide/e2e-tests.md]: K3S, K8S or OpenShift cluster

**Review by hikarukin — CHANGES_REQUESTED** [review]
Some minor textual changes requested.

File: docs/development-guide/e2e-tests.md (hunk 2 above)
Comment (portly-halicore-76) [reply docs/development-guide/e2e-tests.md]: Vanilla k8s can be any of Kind, k3s, minikube etc. No need to specify right?

**Review by AxiomaticFixedChimpanzee — APPROVED** [review]

**Review by hikarukin — APPROVED** [review]

**Pull request merged**